src/universal/modules/outcomeCard/containers/OutcomeCard/OutcomeCardContainer.js

```diff
@@ -72,6 +85,19 @@ class OutcomeCardContainer extends Component {
     });
   };
 
+  isEditing = () => {
+    return !this.state.activeEditingClients.isEmpty()
+  }
+
+  trackEditingClient = (uid, isEditing) => {
+    this.setState((curState) => {
+      const currentClients = curState.activeEditingClients;
```

Comment:
if you see me change the name of `client` before i merge this, it's nothing wrong in your code. In the app I just try to keep the word `client` reserved for websocket clients (I get easily confused).